bigbluebutton-html5/imports/ui/components/chat/chat-graphql/chat-message-list/page/chat-message/component.tsx

```diff
@@ -137,6 +145,19 @@ const ChatMesssage: React.FC<ChatMessageProps> = ({
             />
           ),
         };
+      case ChatMessageType.USER_AWAY_STATUS_MSG:
+        const {user, away} = JSON.parse(message.messageMetadata)
+        return {
+          name: intl.formatMessage(intlMessages.systemLabel),
+          color: '#0F70D7',
+          isModerator: true,
+          component: (
+            <ChatMessageTextContent
+              emphasizedMessage
+              text={user + " " + ((away) ? intl.formatMessage(intlMessages.userAway) : intl.formatMessage(intlMessages.userNotAway))}
```

Comment:
When internationalizing messages, avoid directly concatenating an `intlMessage` with another string, as word order can vary across languages, potentially resulting in grammatical errors or confusion. Instead, utilize placeholders within the `intlMessage` and pass the necessary variables as parameters to `FormattedMessage`. An example in English would be `"app.chat.away": "{0} is away"`, where `{0}` is dynamically replaced.

For maintaining consistency with the existing code style and avoiding changes to the messages, consider the approach used previously. Here's an illustration:

![image](https://github.com/bigbluebutton/bigbluebutton/assets/5660191/b7f2e54e-12b5-4728-b6b3-9a255ba9c90b)

In akka-apps, you can use the `insertSystemMsg` function with the `senderName` parameter to achieve this without altering the message structure.